### ISSUE TITLE
An option should be given to clear the chat

### ISSUE THREAD

**Issue by SudhanvaMG** [fossasia/chat.susi.ai#1166]
**Actual Behaviour**

There is no option for the user to clear the entire chat with SUSI .

**Expected Behaviour**

An optioned should be mentioned in the chat where an user can clear the entire conversation history

**Comment by stealthanthrax** [comment]
I'd like to work on it

**Comment by SudhanvaMG** [comment]
@stealthanthrax  sure .

**Comment by akshatnitd** [comment]
I would like to work on this, as this issue is long unsolved.

**Comment by akshatnitd** [comment]
I can work on it, once https://github.com/fossasia/susi_server/issues/631 is done

**Comment by mariobehling** [comment]
Currently out of scope.
